src/main/java/com/jagrosh/jmusicbot/commands/owner/PlaylistCmd.java

```diff
@@ -169,6 +176,75 @@ protected void execute(CommandEvent event)
             }
         }
     }
+
+    public class AppendlistqCmd extends OwnerCommand 
+    {
+        public AppendlistqCmd()
+        {
+            this.name = "appendq";
+            this.aliases = new String[]{"addq"};
+            this.help = "appends songs in queue to an existing playlist";
+            this.arguments = "<name>";
+            this.guildOnly = false;
+        }
+
+        @Override
+        protected void execute(CommandEvent event) 
+        {
+            String[] parts = event.getArgs().split("\\s+", 2);
+            if(parts.length<1)
+            {
+                event.reply(event.getClient().getError()+" Please include a playlist name to add!");
+                return;
+            }
+            String pname = parts[0];
+            Playlist playlist = bot.getPlaylistLoader().getPlaylist(pname);
+            if(playlist==null)
+                event.reply(event.getClient().getError()+" Playlist `"+pname+"` doesn't exist!");
```

Comment:
Some old code may still use this, but it is preferred to use `event.replySuccess`, `event.replyWarning` or `event.replyError` now.

```java
event.replyError("Playlist `"+pname+"` doesn't exist!");
```

Replace all `event.reply`s across the PR.